xwiki-platform-core/xwiki-platform-invitation/xwiki-platform-invitation-ui/pom.xml

```diff
@@ -36,4 +36,13 @@
     <!-- Name to display by the Extension Manager -->
     <xwiki.extension.name>Invitation Application</xwiki.extension.name>
   </properties>
+  <dependencies>
```

Comment:
maybe provide a comment to explain for what feature of the invitation app this is needed? I guess it's for the config of the invitation app in the Admin UI?